### DISCUSSION TITLE
I propose to make the game mode in which task markers are not displayed on the map.

### DISCUSSION THREAD

**Discussion by pavel4win** [FakeFishGames/Barotrauma#13662] [Ideas and suggestions]
Now barotrauma is a game of “I came, I saw, I won.”
I propose to make the game mode in which task markers are not displayed on the map.
This mode can be enabled as an option in the game or network server settings.
This will encourage players to explore the level and look for the quest item.

For example, if the task is to destroy animals, how can we know where they are now?
All that is known is that they are "somewhere" and we known swarm count.
After all, they don’t have radio collars.
Accordingly, the completion or failure of the mission should be known only after docking.
The same applies to searching for minerals/artifacts.
Perhaps there should be a mark of the cave in which the minerals/artifacts were noticed, but not their exact location.
This also applies to lighthouses and pirate or separatist submarines.
Their exact location should not be shown on the map as an unambiguous mark. All that is known from the assignment is “there is a lighthouse somewhere.” Seek and you will find.

Another suggestion:
Why are all the people encountered during the round enemies?
I suggest doing rescue missions. For example, there may be survivors on a sunken boat, or scientists trapped in a lighthouse. In these missions, it will be necessary to escort the rescued to the sub and deliver them to the station.

**Comment by SomeRandomNoobKekeke** [comment]
Here, try this https://steamcommunity.com/sharedfiles/filedetails/?id=3198868074

Btw it's possible to show sonar labels only on mouse hover

~And i think it's a bad idea to remove minerals marker because if they spawn not in cave then you won't find them~

I figured it out; minerals will only be visible on ship sonar but not on handheld one
Then it will be possible to find them on walls, and you won't see them inside caves from ship sonar, and sonar upgrades won't be useless

**Reply by pavel4win** [reply]
Thank you, this is very useful mod. I will use it on my server.

**Reply by pavel4win** [reply]
Quest minerals are not visible on sonar at all. That is, neither by hand nor by ship.
They are not displayed as a regular diamond icon.
Therefore, if the quest minerals are in a cave, it is not a problem to search the entire cave. But if minerals are on the surface, they can only be detected visually through periscopes.
